tests/e2e/run-e2e.sh

```
@@ -88,125 +99,133 @@ if [ ! -d "$DIST_HOME" ]; then
+        --data-urlencode "client_id=CONSOLE" \
+        --data-urlencode "redirect_uri=$CONSOLE_REDIRECT_URI" \
+        --data-urlencode "scope=system" \
+        --data-urlencode "resource=$SERVER_URL/mcp" \
+        --data-urlencode "response_type=code" \
+        --data-urlencode "code_challenge=$CODE_CHALLENGE" \
+        --data-urlencode "code_challenge_method=S256"
+
+    local LOCATION AUTH_ID EXEC_ID
+    LOCATION=$(grep -i "^location:" "$headers_file" | tr -d '\r' | sed 's/^[Ll]ocation: //')
+    rm -f "$headers_file"
+    AUTH_ID=$(echo "$LOCATION" | sed 's/.*[?&]authId=\([^&]*\).*/\1/')
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)